src/main.rs

```diff
@@ -164,9 +179,10 @@ async fn main() -> Result<(), Box<dyn Error>> {
                         let exposed_settings = SseServerSettings {
                             bind_addr: bind_addr.parse::<SocketAddr>()?,
                             keep_alive: Some(Duration::from_secs(15)),
```

Comment:
Do we need to keep the same duration as in line 137?

Reply:
I will revert Line 137, thanks.